packages/@o3r/design/src/core/design-token/parsers/design-token.parser.ts

```diff
@@ -30,11 +31,37 @@ const getExtensions = (nodes: NodeReference[], context: DesignTokenContext | und
   }, {} as DesignTokenGroupExtensions & DesignTokenExtensions);
 };
 const getReferences = (cssRawValue: string) => Array.from(cssRawValue.matchAll(tokenReferenceRegExp)).map(([,tokenRef]) => tokenRef);
+const applyConversion = (token: DesignTokenVariableStructure, value: string) => {
+  if (typeof token.extensions.o3rUnit === 'undefined' && typeof token.extensions.o3rRatio === 'undefined') {
+    return value;
+  }
+
+  const splitValue = splitValueNumericRegExp.exec(value);
+  if (!splitValue) {
+    return value;
+  }
+
+  let newValue = value;
+  if (typeof token.extensions.o3rRatio !== 'undefined') {
+    newValue = newValue.replace(splitValue[1], (parseFloat(splitValue[1]) * token.extensions.o3rRatio).toString());
```

Comment:
Are we sure we want to do multiplication like that in javascript ?
![image](https://github.com/user-attachments/assets/33c2654c-bf5d-4c35-a786-ad159dcfcc33)